service/src/main/kotlin/xyz/lebkuchenfm/domain/commands/processors/XCommandProcessor.kt

```diff
@@ -0,0 +1,38 @@
+package xyz.lebkuchenfm.domain.commands.processors
+
+import io.github.oshai.kotlinlogging.KotlinLogging
+import xyz.lebkuchenfm.domain.commands.CommandParameters
+import xyz.lebkuchenfm.domain.commands.CommandProcessor
+import xyz.lebkuchenfm.domain.commands.model.Command
+import xyz.lebkuchenfm.domain.commands.model.CommandProcessingResult
+import xyz.lebkuchenfm.domain.eventstream.EventStream
+import xyz.lebkuchenfm.domain.eventstream.PlayXSoundEvent
+import xyz.lebkuchenfm.domain.xsounds.XSoundsService
+
+private val logger = KotlinLogging.logger {}
+
+class XCommandProcessor(private val xSoundsService: XSoundsService, private val eventStream: EventStream) :
+    CommandProcessor(
+        key = "x",
+        shortKey = null,
+        helpMessage = "Puszcza szalony dźwięk!",
+        exampleUsages = listOf("airhorn"),
+        parameters = CommandParameters(
+            parameters = listOf(
+                CommandParameters.RequiredCommandParameter("sound-name"),
+            ),
+        ),
+    ) {
+    override fun execute(command: Command): CommandProcessingResult {
+        val soundName = command.rawArgs
+            ?: return error("You have to provide sound name.", logger)
```

Comment:
So we have user facing error in english but help message in polish? 🤔

Reply:
Oh yes, I wanted to talk to you about it and forgot. First - it should be consistent, but I forgot to change it.

My current thinking is that we will write everything in english (as opposed to polish-english that we currently have on prod) and then tackle i18n after v3 is deployed. How does that sound to you?